modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// *
+:_module-type: PROCEDURE
+[id="ipi-verifying-nodes-after-installation_{context}"]
+= Verifying node state after installation
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
+Your OpenShift Container Platform installation completes when the following installation health checks are successful:
+
+* The provisioning host can access the OpenShift Web Console.
+
+* All master nodes are ready.
+
+* All cluster Operators are available.
+
+However, although your installation successfully completes, not all worker nodes in the cluster are instantly provisioned. Some worker nodes remain in a non-provisioned state, such as NotReady, Inspecting or Provisioned. 
```

Comment:
```suggestion
[NOTE]
====
It can take some time after cluster installation is successfully completed before all all worker nodes report as `READY`. The Cluster Operator continuously attempts to provision worker nodes for a set period of time. The state of a worker node only changes to `DEGRADED` when the Cluster Operator fails to provision the worker node within this time period. You can define this time period in the `install-config.yaml` file.
==== 
```
1. Operator names take title case. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#documentation-guidelines
2. Operators are discrete pieces of software that are deployed separately and run in the cluster. Distinct from node/machine/host.
2. Use backticks to mark any code or text from the CLI.

Q: what is the exact param that you config this time-out value in `install-config.yaml`?

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references